main.go

```diff
@@ -117,6 +118,13 @@ func main() {
 				tw.AppendRow(table.Row{service, ac.FormatMoney(amounts[0]), ac.FormatMoney(amounts[1]), ac.FormatMoney(amounts[2])})
 			}
 			tw.AppendFooter(table.Row{"Total", ac.FormatMoney(totalAmount), ac.FormatMoney(totalSecondAmount), ac.FormatMoney(totalDelta)})
+
+			tw.SetColumnConfigs([]table.ColumnConfig{
+				{Name: firstMonthStart, Align: text.AlignRight},
+				{Name: secondMonthHeader, Align: text.AlignRight},
+				{Name: "Delta", Align: text.AlignRight},
```

Comment:
Add `AlignFooter: text.AlignRight` as well to these. That should fix the issue you're seeing with it just being on the body of the data.